frontend/src/components/MonacoEditor/MonacoEditor.vue

```diff
@@ -175,15 +177,24 @@ const init = async () => {
 
   // when editor change selection, emit change-selection event with selected text
   editorInstance.onDidChangeCursorSelection((e) => {
-    const selectedText = editorInstance.getModel()?.getValueInRange({
-      startLineNumber: e.selection.startLineNumber,
-      startColumn: e.selection.startColumn,
-      endLineNumber: e.selection.endLineNumber,
-      endColumn: e.selection.endColumn,
-    }) as string;
+    const selectedText = editorInstance
+      .getModel()
+      ?.getValueInRange(e.selection) as string;
     emit("change-selection", selectedText);
   });
 
+  editorInstance.onDidChangeCursorPosition((e) => {
+    const { defineLineDecorations, disposeLineDecorations } =
+      useLineDecorations(editorInstance, e.position);
```

Comment:
nit: The indention seems a bit off

Reply:
> nit: The indention seems a bit off

The 187 lines are longer than 80 characters, so the formatter automatically wrapped this line.